app/src/main/kotlin/at/bitfire/davdroid/sync/JtxSyncer.kt

```diff
@@ -76,10 +77,18 @@ class JtxSyncer @AssistedInject constructor(
         localCollection.updateCollection(remoteCollection, owner, accountSettings.getManageCalendarColors())
     }
 
-    override fun create(provider: ContentProviderClient, remoteCollection: Collection) {
+    override fun create(provider: ContentProviderClient, remoteCollection: Collection): LocalJtxCollection {
         logger.log(Level.INFO, "Adding local jtx collection", remoteCollection)
         val owner = remoteCollection.ownerId?.let { principalRepository.get(it) }
         LocalJtxCollection.create(account, provider, remoteCollection, owner)
+        return JtxCollection.find(
+            account,
+            provider,
+            context,
+            LocalJtxCollection.Factory,
+            "${JtxContract.JtxCollection.URL} = ?",
+            arrayOf("${remoteCollection.url}")
+        ).first()
```

Comment:
Is there a reason not to use the ID, like for the other collections?

`JtxCollection.create` returns an `Uri`, can't we use it to get the ID similar to the other collection types?

Reply:
Ah no, I overlooked it, since `LocalJtxCollection.create()` did not return it. Will change!